### ISSUE TITLE
addLibraryType() typo

### ISSUE THREAD

**Issue by sklages** [TGAC/miso-lims#1394]
`addLibraryType()` contains a smal typo:

```
CREATE DEFINER=`root`@`localhost` PROCEDURE `addLibraryType`(
  iDescription varchar(255),
  iPlatformType varchar(50),
  archived tinyint(1)
)
```
should state 
```
CREATE DEFINER=`root`@`localhost` PROCEDURE `addLibraryType`(
  iDescription varchar(255),
  iPlatformType varchar(50),
  iArchived tinyint(1)
)
```

`archived` --> `iArchived`

**Comment by djcooke** [comment]
Thanks Sven. Will fix for the next release

**Comment by djcooke** [comment]
Fixed in #1395
